bootstrap.sh

```diff
@@ -4,13 +4,23 @@ set -euo pipefail
 SECRETS=$(echo $VCAP_SERVICES | jq -r '.["user-provided"][] | select(.name == "secrets") | .credentials')
 APP_NAME=$(echo $VCAP_APPLICATION | jq -r '.name')
 APP_ROOT=$(dirname "${BASH_SOURCE[0]}")
+APP_ID=$(echo "$VCAP_APPLICATION" | jq -r '.application_id')
 
-S3_BUCKET=$(echo $VCAP_SERVICES | jq -r '.["s3"][]? | select(.name == "storage") | .credentials.bucket')
-S3_REGION=$(echo $VCAP_SERVICES | jq -r '.["s3"][]? | select(.name == "storage") | .credentials.region')
+DB_NAME=$(echo $VCAP_SERVICES | jq -r '.["aws-rds"][] | .credentials.db_name')
+DB_USER=$(echo $VCAP_SERVICES | jq -r '.["aws-rds"][] | .credentials.username')
+DB_PW=$(echo $VCAP_SERVICES | jq -r '.["aws-rds"][] | .credentials.password')
+DB_HOST=$(echo $VCAP_SERVICES | jq -r '.["aws-rds"][] | .credentials.host')
+DB_PORT=$(echo $VCAP_SERVICES | jq -r '.["aws-rds"][] | .credentials.port')
+
+S3_BUCKET=$(echo "$VCAP_SERVICES" | jq -r '.["s3"][]? | select(.name == "storage") | .credentials.bucket')
```

Comment:
Lines 15-18 should appear near the top of .profile, to avoid the need to use an S3 service key and set environment variables on the app.

Reply:
My reading of the docs indicate that what happens in .profile only happens when it's building the image, and the environment variables that it sets don't stick.  Is that not true?

Reply:
Nope, `.profile` is a great place to map between CF-isms like `VCAP_SERVICES` and `VCAP_APPLICATION` env vars and any local file-based config.

Reply:
(I can see that the docs don't make that clear enough... I'll send them a PR to clarify it.)

Reply:
Here's where it happens btw:
https://github.com/cloudfoundry/buildpackapplifecycle/blob/master/launcher/launcher_unix.go#L25-L27

Reply:
As learned through the school of hard knocks: 
https://cloudfoundry.slack.com/archives/C02HWMDUQ/p1559711709000700?thread_ts=1559693364.001900&cid=C02HWMDUQ

Reply:
Hmm.  I think you are right, but I'd like to address this in a separate PR because this works, people are waiting for it, and I don't want to lose @fureigh 's attention again.  :-)

Reply:
Let's just get it merged and I'll take care of it.

Reply:
Handled in https://github.com/18F/cf-ex-drupal8/pull/31